test/e2e-v2/cases/redis/redis-exporter/scripts/crontable.txt

```diff
@@ -0,0 +1,2 @@
+* * * * * /scripts/slowlog.sh
```

Comment:
I think this matters as well. Let's polish the docs to indicate we need to scratch logs periodically through fluentbit, and the scripts in the e2e are demo only.